GitUI/UserEnvironmentInformation.cs

```diff
@@ -67,6 +69,48 @@ public static string GetGitVersionInfo(string? gitVersion, GitVersion lastSuppor
             return gitVersion;
         }
 
+        public static string GetDotnetVersionInfo()
+        {
+            StringBuilder sb = new();
+            Executable dotnet = new(DOTNET_CMD);
+            ArgumentString[] args = new ArgumentString[]
+           {
+                new ArgumentBuilder()
+                {
+                    "--list-runtimes"
+                },
+
+                new ArgumentBuilder()
+                {
+                    "--list-sdks"
+                }
+           };
```

Comment:
nit: indent; `string[]` would have worked, too, wouldn't it?

Reply:
Yes but they are arguments and not strictly string arrays and I'm making it so if there is another command with multiple arguments, it's easier to manage and update.